CMakeLists.txt

```diff
@@ -114,6 +114,10 @@ endif(MSVC)
 # Dependencies setup
 ########################################################################
 
+# cpu_features
+set(BUILD_PIC ON)
+add_subdirectory(cpu_features)
```

Comment:
This seems to be the offending part for all the PIC issues.
If you add a message here:
```cmake
set(BUILD_PIC ON)
message(STATUS "pic='${BUILD_PIC}'")
add_subdirectory(cpu_features)
```
It'll print `pic=ON`. But CMake will ignore this setting in the subdirectory `cpu_features`. There, this variable is unset again and consequently let's the build fail. If you run cmake with `-DBUILD_PIC=ON` it'll work. This is a really frustrating experience at the moment. 
@noc0lour, do you have any suggestion how to fix this?

Reply:
`CACHE` it. That's what the `-D` setting is, for all practical purposes. That way the variable is set in any subdirectory that might reference it, unless it has been explicitly removed from the cache.

Reply:
Thanks. That works. It did work without cache on my machine though.